backend/benefit/messages/tests/test_api.py

```diff
@@ -228,27 +228,36 @@ def test_create_handler_message_invalid(handler_api_client, handling_application
     )
 
 
-def test_handler_must_message_first(
+@pytest.mark.parametrize(
+    "status,expected_result",
+    [
+        (ApplicationStatus.RECEIVED, 201),
+        (ApplicationStatus.HANDLING, 201),
+        (ApplicationStatus.ADDITIONAL_INFORMATION_NEEDED, 201),
+        (ApplicationStatus.ACCEPTED, 400),
+        (ApplicationStatus.REJECTED, 400),
+        (ApplicationStatus.CANCELLED, 400),
+    ],
```

Comment:
Would it make sense to have `(ApplicationStatus.DRAFT, 400),` in this list?

Reply:
good point, fixed